terraform/k8s/mojaloop-stateful-resources.json

```diff
@@ -579,60 +579,68 @@
             "kafka_provisioning": {
                 "enabled": true,
                 "topics": [
+                    {
+                      "name": "topic-event",
+                      "partitions": 3,
+                      "replicationFactor": 3
+                    },
                     {
                       "name": "topic-transfer-prepare",
                       "partitions": 4,
-                      "replicationFactor": 1
+                      "replicationFactor": 3
                     },
                     {
                       "name": "topic-transfer-position",
                       "partitions": 25,
-                      "replicationFactor": 1
+                      "replicationFactor": 3
                     },
                     {
                       "name": "topic-transfer-position-batch",
                       "partitions": 4,
-                      "replicationFactor": 1
+                      "replicationFactor": 3
                     },
                     {
                       "name": "topic-transfer-fulfil",
                       "partitions": 4,
-                      "replicationFactor": 1
+                      "replicationFactor": 3
                     },
                     {
                       "name": "topic-notification-event",
                       "partitions": 4,
-                      "replicationFactor": 1
+                      "replicationFactor": 3,
+                      "config": {
+                        "retention.bytes": "20000000000"
```

Comment:
We need to set retention for topic-event and not for topic-notification-event.

Reply:
of course...

multitasking error